### PR TITLE
Add a simple Runbook

### PR DESCRIPTION
**Pull request by 46bit**
What
----

This commit adds a simple document (a "runbook") describing how to operate `paas-auditor`. The goal is to share key knowledge from my head while we're trialling this in production.

How to review
-----

Read the commit. Please complain if you have important lingering questions.

Who can review
-----

Not @46bit.

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Dealing with issues
+
+### It's OK to stop it
+
+Cloud Controller stores Audit Events for about 31 days. If Cloud Controller is experiencing high load you are absolutely fine to stop it.
+
+### How to stop it
+
+It's typically in the `admin` org's `billing` space. Straightforwardly cf stop` the app:
```
Comment (tlwr) [inline RUNBOOK.md]:
```suggestion
It's typically in the `admin` org's `billing` space. Straightforwardly `cf stop` the app:
```

File: RUNBOOK.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+* For your `cf` CLI to be logged into Cloud Foundry;
+* For the `DEPLOY_ENV` environment variable to be set appropriately;
+* For your AWS access to be configured so the `aws` CLI will work.
+* To have already created a Postgres service named `auditor-db`.
+
+```
+cd paas-auditor
+cf target -o admin -s billing
+cf push --no-start
+cf bind-service paas-auditor auditor-db
```
Comment (tlwr) [inline RUNBOOK.md]: Thoughts on this going in the manifest (separate PR)

File: RUNBOOK.md (hunk 1 above)
Comment (46bit) [reply RUNBOOK.md]: Fixed – cheers :)

File: RUNBOOK.md (hunk 2 above)
Comment (46bit) [reply RUNBOOK.md]: Go for it

File: RUNBOOK.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* If your database already has events stored, it will fetch data since the most recent event. To ensure nothing is missed, it actually fetches data from 5 minutes before then.
+
+Once that initial fetching finishes, it will wake up to bring itself up to date every 5 minutes.
+
+### How to observe it working
+
+You should be able to see quite descriptive logs from:
+
+```
+cf logs audit-db
```
Comment (tlwr) [inline RUNBOOK.md]:
```suggestion
cf logs paas-auditor
```

Comment (46bit) [reply RUNBOOK.md]: Fixed :)

**Pull request merged**